telegram_export/utils.py

```diff
@@ -211,3 +213,38 @@ def action_to_name(action):
         types.ChannelAdminLogEventActionToggleSignatures: 'toggle.signatures',
         types.ChannelAdminLogEventActionUpdatePinned: 'update.pinned',
     }.get(type(action), None)
+
+
+def parse_proxy_str(proxy_str):
+    """
+    Returns proxy from given string
+    """
+    url_parser = urlparse(proxy_str)
+    proxy_type = None
+    proxy_type_str = url_parser.scheme
+    
+    if proxy_type_str.lower() == "socks5":
+        proxy_type = socks.SOCKS5
+    elif proxy_type_str.lower() == "socks4":
+        proxy_type = socks.SOCKS4
+    elif proxy_type_str.lower() == "https":
+        proxy_type = socks.HTTP
+    elif proxy_type_str.lower() == "http":
+        proxy_type = socks.HTTP
+    else:
+        return None
```

Comment:
Like @expectocode suggested, `raise ValueError(...)` may be more appropriated.

Reply:
Thanks, for good comments